.github/workflows/ci.yaml

```diff
@@ -36,8 +33,27 @@ jobs:
             contracts/precompiles/artifacts
             bootloader/build
 
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.18.0
+          cache: yarn
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Run lint
+        run: yarn lint
```

Comment:
does it fail when there are lint issues? or does it fix them?

Reply:
It just fails